### PR TITLE
Blog post for fulcio CI identity provider onboarding

### PR DESCRIPTION
**Pull request by javanlacerda**

### CODE REVIEW

File: content/fulcio-ci-provider.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
++++
+title = "Fulcio Streamlines Onboarding for Identity Providers, mainly CI providers"
```
Comment (haydentherapper) [inline content/fulcio-ci-provider.md:2]:
```suggestion
title = "Fulcio Streamlines Onboarding for CI Identity Providers"
```

**Review by haydentherapper — APPROVED** [review]
Thanks!

**Review by bobcallaway — APPROVED** [review]

**Review by haydentherapper — APPROVED** [review]

**Review by SantiagoTorres — APPROVED** [review]

**Review by lukehinds — APPROVED** [review]

File: content/fulcio-ci-provider.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
++++
+title = "Fulcio Streamlines Onboarding for CI Identity Providers"
+author = "Javan Lacerda (@javanlacerda)"
+date = "2024-08-26"
```
Comment (bobcallaway) [inline content/fulcio-ci-provider.md:4]: Can you update this date to today and I’ll merge? Thanks!

Comment (javanlacerda) [reply content/fulcio-ci-provider.md]: Done!

**Pull request merged**